src/pages/post/index.js

```diff
@@ -226,9 +226,12 @@ class Post extends Component {
                   <span>
                     <Icon type="calendar" style={{ marginRight: "0.5rem" }} />
                     发布于&nbsp;
-                    {formatDistanceToNow(new Date(post.created_at), {
-                      locale: chinese
-                    })}
+                    {formatDistanceToNow(
+                      new Date(post && post.created_at ? post.created_at : 0),
```

Comment:
Thank you very much for your PR to fix the bugs for me.

But this line seem doesn't not need at all. ref: [deepscan](https://deepscan.io/dashboard/#view=project&tid=5773&pid=7588&bid=79785&subview=pull-request&prid=178232)

Reply:
It should be 

```jsx
{ post.created_at ? formatDistanceToNow(post.created_at) : '' }
```

Reply:
@axetroy Ok, it's has been removed _(•̀ω•́,